Tidy up root directory

neilweidinger opened the pull request:
## Description
Moved vscode config and main logo txt file to `utils` directory. Also removed unused setup script, that was originally used to initialize the Eigen submodule but now not needed as Eigen is already included in the repo.

mkristien commented:
The workspace file has to remain in the root directory. First, it is currently configured with some relative paths assuming root directory. Second, being in the root directory allows people to easily open the workspace by `hyped2020> code hyped.core-workspace`

Furthermore, `setup.sh` I want to keep (as discussed [before](https://github.com/Hyp-ed/hyped-2020/pull/1)). In fact, the file is used by one of the open [PR](https://github.com/Hyp-ed/hyped-2020/pull/9).

neilweidinger commented:
My bad about the vscode workspace file, should have checked before I moved it! I removed `setup.sh` because it hadn't been worked on in two weeks, thought nobody was using it.
